docs/.eleventy/layouts/base.njk

```diff
@@ -123,6 +123,13 @@ html, body, .markdown-body {
   margin: auto;
   opacity: .6;
 }
+
+.header-snowpack-blurb {
```

Comment:
style nits: okay, so I tried this out in the browser with `text-align: justify` and I think it looked really good!